### ISSUE TITLE
cannot parse arrow function with parameters

### ISSUE THREAD

**Issue by mobljs** [cherow/cherow#232]
this without parameters can work : 
cherow.parse(`
class Some {
		render=( )=>{
			return null;
		}
}`);

but this can not :
cherow.parse(`
class Some {
		render=(a,b)=>{
			return null;
		}
}`);

**Comment by KFlash** [comment]
what's the error message you got?

**Comment by KFlash** [comment]
It will be solved in 2.0

**Comment by joaomilho** [comment]
The same goes when you have the lambda in an export default:

```
> cherow.parse('export default 1', {next:true, module:true})
{ type: 'Program',
  sourceType: 'module',
  body: [ { type: 'ExportDefaultDeclaration', declaration: [Object] } ] }

> cherow.parse('export default () => {}', {next:true, module:true})
{ type: 'Program',
  sourceType: 'module',
  body: [ { type: 'ExportDefaultDeclaration', declaration: [Object] } ] }

> cherow.parse('export default (a,b) => {}', {next:true, module:true})
SyntaxError: Line 1, column 21: Invalid binding pattern
```

**Comment by KFlash** [comment]
This is an 1.6.8 bug. Will be addressed in 2.0

**Comment by KFlash** [comment]
Cherow 2.0 soon to be shipped, and this issue have been confirmed solved in the upcoming v. 2.0.
